txn_test.go

```diff
@@ -286,7 +289,11 @@ func (s *txnSuite) TestExcessiveContention(c *gc.C) {
 	}
 	err := s.txnRunner.Run(buildTxn)
 	c.Assert(err, gc.Equals, jujutxn.ErrExcessiveContention)
-	c.Assert(maxAttempt, gc.Equals, 2)
+	if s.supportsSST {
+		c.Assert(maxAttempt, gc.Equals, 19)
```

Comment:
Where does the magic number 19 come from?

Reply:
it's 20 (retry count for sstxn) minus 1

Reply:
just as 2 is (3 minus 1) fr client side txn